src/middlewares/errorMiddleware.js

```diff
@@ -0,0 +1,19 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { ConsoleLogger } from '../untils/consoleLogger.js';
+
+export const errorMiddleware = (error, req, res, next) => {
+  ConsoleLogger.error('Error: ', error);
+
+  if (error instanceof ApiError) {
+    return res.status(error.status).send({
+      message: error.message,
+      errors: error.errors,
+    });
+  }
+
+  res.statusCode = 500;
```

Comment:
Instead of directly setting `res.statusCode`, use `res.status(500)` for consistency with the rest of the response handling.